app/models/concerns/servers/command.rb

```diff
@@ -98,4 +95,68 @@ def execute_remote run, script_path
         output: out.to_s + status_text.to_s
       }
     end
+
+    def local_execution execution, script_path
+      status = 1
+
+      Open3.popen2e rails, 'runner', script_path do |stdin, stdout, thread|
+        execution.update! pid: thread.pid
+
+        stdout.each { |line| execution.new_line line }
+
+        status = thread.value.exitstatus.to_i
+      end
+
+      status
+    end
+
+    def remote_execution execution, script_path
+      status = 1
+
+      Net::SSH.start hostname, user, ssh_options  do |ssh|
+        # execution permission
+        ssh.exec! "chmod +x #{script_path}"
+
+        output_proc = proc { |data| execution.new_line data }
```

Comment:
```suggestion
        output_proc = -> (data) { execution.new_line data }
```

Lo amás en silencio.

Reply:
jaja si lo amo, le puse proc porque ya le habia puesto proc  a la variable jaja.